components/tools/OmeroWeb/omeroweb/webgateway/templates/webgateway/viewport/omero_image.html

```diff
@@ -1066,6 +1066,10 @@ <h3 class="can-collapse" id="bulk-annotations" style="display: none;">Bulk Annot
       var disable_intmax = (viewport.loadedImg.rdefs.invertAxis || viewport.getSizes().z < 2);
       $('[name="wblitz-proj"][value=intmax]').prop('disabled', disable_intmax);
 
+      // Turn off interpolation if disabled
+      {% if not interpolate %}
+        viewport.setPixelated(true);
```

Comment:
instead of adding interpolation to template variable it would be preferred to add it to ``viewport.loadedImg``